examples/typescript/your_fungible_asset.ts

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable no-console */
+/* eslint-disable max-len */
+
+import {
+  Account,
+  AccountAddress,
+  Aptos,
+  AptosConfig,
+  InputViewRequestData,
+  Network,
+  NetworkToNetworkName,
+} from "@aptos-labs/ts-sdk";
+import { compilePackage, getPackageBytesToPublish } from "./utils";
+/**
+ * This example demonstrate how one can compile, deploy, and mint its own fungible asset (FA)
+ * It uses the fa_coin.move module that can be found in the move folder
+ *
+ * Before running this example, we should compile the package locally:
+ * 1. Acquire the Aptos CLI, see https://aptos.dev/cli-tools/aptos-cli/use-cli/install-aptos-cli
+ * 2. cd `~/aptos-ts-sdk/examples/typescript`
+ * 3. Run `pnpm run your_coin`
+ */
+
+// Setup the client
+const APTOS_NETWORK: Network = NetworkToNetworkName[process.env.APTOS_NETWORK] || Network.TESTNET;
+const config = new AptosConfig({
+  network: APTOS_NETWORK,
+  clientConfig: { AUTH_TOKEN: "ejfklsdfj7vr4388fhhfh3f78hf345345hf00da0" },
```

Comment:
I think, having an auth token built into the examples, defeats the purpose if everyone knows it?  This and any other auth tokens we have put out will probably need to be revoked.

Reply:
oh wow, that shouldn't be here